docs/DeveloperGuide.adoc

```diff
@@ -279,51 +279,107 @@ Refer to the guide <<DevOps#, here>>.
 
 *Target user profile*:
 
-* has a need to manage a significant number of contacts
+* a librarian in a small town library that has to serve many library users (borrowers) quickly
+* has a need to manage a significant number of books and borrowers
 * prefer desktop apps over other types
 * can type fast
 * prefers typing over mouse input
 * is reasonably comfortable using CLI apps
 
-*Value proposition*: manage contacts faster than a typical mouse/GUI driven app
+*Value proposition*: Many people visit the neighborhood library to borrow books and also donate their books. There is
+always a long queue in this small library and the librarian would have to type quickly to handle the long queue. Liberry
+can manage a library system faster than a typical mouse/GUI driven app.
 
 [appendix]
 == User Stories
 
 Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (unlikely to have) - `*`
 
-[width="59%",cols="22%,<23%,<25%,<30%",options="header",]
+[width="75%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|`* * *` |librarian |add a book brought/donated by people to the library |keep track of the addition of the book
 
-|`* * *` |user |add a new person |
+|`* * *` |librarian |delete books that are no longer available |keep track of the deletion of the book
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |helpful librarian |search for certain book by the title/author/genre |help borrowers check if it is
+available
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |forgetful librarian |mark a book as loaned out |tell borrowers if the book is available
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* * *` |forgetful librarian |mark a book as returned |let borrowers know that the book will now be available
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
-|=======================================================================
+|`* * *` |librarian |generate a list of overdue books and their borrowers |know which borrower has overdue books
+and which books are overdue
 
-_{More to be added}_
+|`* * *` |librarian |generate a list of currently loaned / available books |do inventory checks
+
+|`* * *` |meticulous librarian |record the movement of books in and out |keep track of available books here
+
+|`* * *` |helpful librarian |assist a borrower to registering in the system |help new borrowers start borrowing books
```

Comment:
It is unclear if the librarian is helping the borrower with the process of registering or if the librarian is registering the borrower himself

Reply:
User story is not the same as implementation. More details will be provided in the implementation/use case.